it/test/uk/gov/hmrc/tradergoodsprofilesrouter/UpdateRecordIntegrationSpec.scala

```diff
@@ -628,19 +628,35 @@ class UpdateRecordIntegrationSpec extends HawkIntegrationSpec with AuthTestSuppo
             .futureValue
 
           response.status shouldBe BAD_REQUEST
-          response.json   shouldBe Json.obj(
-            "correlationId" -> correlationId,
-            "code"          -> "BAD_REQUEST",
-            "message"       -> "Bad Request",
-            "errors"        -> Json.arr(
-              Json.obj(
-                "code"        -> "INVALID_HEADER",
-                "message"     -> "Missing mandatory header X-Client-ID",
-                "errorNumber" -> 6000
+          if (!appConfig.isClientIdHeaderDisabled) {
```

Comment:
we do not need to if statement as isClientIdHeaderDisabled will always be false in the test

Reply:
Removed the if statement